### DISCUSSION TITLE
How to setting avalonia default style when we define custom style for controls

### DISCUSSION THREAD

**Discussion by chancezheng** [AvaloniaUI/Avalonia#16516] [Q&A] [answered]
it is seem as 'Style={x:null}' in wpf, i use 'Theme={x:null}' in avalonia is not effect, how to setting up the default style?

**Comment by rabbitism** [comment]
Usually you just need to set `Theme={StaticResource {x:Type YourControlType}}` (or dynamic resource)

**Reply by chancezheng** [reply, marked as answer]
我开始使用controltheme重写控件样式，后变为Style方式引用，再使用Theme={x:null}可以了，谢谢😊
